src/index.html

```diff
@@ -22,6 +22,298 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="./images/logo.png"
+        alt="Moyo Logo"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__link">Samsung</li>
+          <li class="nav__link">Smartphones</li>
+          <li
+            class="nav__link"
+            data-qa="nav-hover"
+          >
+            Laptops & Computers
+          </li>
+          <li class="nav__link">Gadgets</li>
+          <li class="nav__link">Tablets</li>
+          <li class="nav__link">Photo</li>
+          <li class="nav__link">Video</li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star"></div>
+          </div>
```

Comment:
Fix it everywhere
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
```

Reply:
<img width="594" alt="image" src="https://github.com/user-attachments/assets/dc6bc7d0-f74f-4e56-8b31-aebe212e9096" />